sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1010,7 +1031,7 @@ be immediately stale, any code which relies on queue waits should take care
 to ensure waits are not performed on queues in recording mode. For example, by
 using separate queues for graph recording and normal queue operations.
 
-=== Storage Lifetimes
+=== Storage Lifetimes [[storage-lifetimes]]
```

Comment:
shouldn't need the extra `[[storage-lifetimes]]` in order to reference earler with `<<storage-lifetimes, storage lifetimes>>`

Reply:
That makes sense from an editorial maintenance point-of-view